src/plugins/validation/oas3/semantic-validators/security-definitions-ibm.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2019 IBM All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Assertation 1: `type` is a necessary property and has four possible values: `apiKey`, `HTTP`, `oauth2`, `openIdConnect`
+// Assertation 2: name property is required for `apiKey` type
+// Assertation 3: `in` property is required for `apiKey` type, valid values are: `query`, `header` or `cookie`
+// Assertation 4: `scheme` property` is required for `http` type
+// Assertation 5: `flows` object is required for `oauth2` type
+// Assertation 6: `opedIdConnectUrl` property is required for `openIdConnect` and must be a valid url
+
```

Comment:
This also was intentional, I think. Currently, we only document "named" (aka configurable) rules in the README - many of the legacy, non-configurable rules from Swagger Editor are not documented. The reason being that they are inherent to the spec and are sort of treated as "assumed checks". I am definitely open to documenting these though

Reply:
I'm just thinking that it would be nice to point to specific section of the spec that motivated a particular check / error message. We don't need to do that in this PR, but as a service to our users I think it would be a good practice to adopt.